docs/scos/dev/the-docker-sdk/202204.0/define-parameters-and-secrets-locally.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Define parameters and secrets locally
+description: Learn how to declare parameters and define their values.
+last_updated: July 28, 2022
+template: howto-guide-template
+---
+
+This document describes how to declare parameters and secrets and define their values without exposing them in deploy files.
+
+## Declare parameters and secrets
+
+1. Define parameters and secrets in the needed deploy file as follows:
+
+```yaml
+...
+environment-configuration:
+    secrets:
+        - name: {SECRET_NAME}
+          grant: {PERMISSIONS}
+          bucket: {BUCKET}
+    params:
+        - name: {PARAMETER_NAME}
+          bucket: {BUCKET}
+          default: {DEFAULT_VALUE}
+          grant: {PERMISSIONS}
+...
+```
+| PARAMETER | DESCRIPTION |
+| - | - |
+| name | Unique identifier of a secret or parameter. You will use it when defining its value in `.env.docker.local`. |
+| grant | IAM users' permissions. `limited` provides read while `public` provides read and write permissions. The default one is `limited`. |
```

Comment:
i guess "IAM" is irrelevant here. 
it's just permission from the customer's prospective.